treeherder/perf/outcome_checker.py

```diff
@@ -0,0 +1,68 @@
+from django.db.models import Q, QuerySet
+
+from enum import Enum
+from typing import List
+
+from treeherder.model.models import Job, Push
+from treeherder.perf.models import (
+    BackfillRecord,
+)
+
+class OutcomeStatus(Enum):
+    SUCCESSFUL = 1
+    FAILED = 2
+    IN_PROGRESS = 3
+
+#TODO delete this after soft launch lands (job_type will come from BackfillRecord)
+def _get_job_type(record):
+    record_context = record.get_context()
+    job_type = None
+
+    if len(record_context) >= 1:
+        job_id = record_context[0]["job_id"]
+        job = Job.objects.get(id=job_id)
+        if job:
+            job_type = job.job_type
+
+    return job_type
+
+
+class OutcomeChecker:
+    """
+    Checks outcome of backfills
+    """
+
+    def __init__(self):
+        pass
+
+    def check(self, record: BackfillRecord) -> int:
```

Comment:
Note: this function doesn't return an `int`, but an enum value.
PyCharm is complaining about this.